pkg/cmd/auth/login/login.go

```diff
@@ -222,7 +226,7 @@ func loginRun(opts *LoginOptions) error {
 }
 
 func promptForHostname(opts *LoginOptions) (string, error) {
-	options := []string{"GitHub.com", "GitHub Enterprise Server"}
+	options := []string{"GitHub.com", "other"}
```

Comment:
Should we capitalize `other`? 

I know the A/C has it uncapitalized, but it seems like there might be some precedent established to capitalize the first letter of prompt options. Example:

```
? How would you like to authenticate GitHub CLI?  [Use arrows to move, type to filter]
> Login with a web browser
  Paste an authentication token
```

[I checked Primer as well](https://primer.style/native/cli/getting-started#make-it-feel-like-github), and that seems to suggest we should use sentence case where we can:

![image](https://github.com/user-attachments/assets/a9c6cbd8-8fef-412d-a9ec-5da82844c7f0)

[This Primer example](https://primer.style/native/cli/components#radio-select) for a radio prompt also uses sentence case:

![image](https://github.com/user-attachments/assets/ad570f06-fb61-42f6-a700-312023ba2c77)